fast64_internal/oot/scene/exporter/to_c/spec.py

```diff
@@ -101,7 +101,7 @@ def editSpecFile(scene: OOTScene, exportInfo: ExportInfo, sceneC: OOTSceneC):
 
                 if sceneC.sceneCutscenesIsUsed():
                     for i in range(len(sceneC.sceneCutscenesC)):
-                        sceneSegInclude += indent + f'include "{includeDir}_cs_{i}.o"\n'
+                        sceneSegInclude += indent + f'include "{includeDir}_scene_cs_{i}.o"\n'
```

Comment:
that's weird, it looks like currently it should be correct:
```c
# ootExportSceneToC
os.path.join(levelPath, scene.sceneName() + "_cs_" + str(i) + ".c"),

# editSpecFile
includeDir = f"build/{getSceneDirFromLevelName(sceneName)}/{sceneName}"
...
sceneSegInclude += indent + f'include "{includeDir}_cs_{i}.o"\n'
```

I guess `scene.sceneName()` has `_scene` and `sceneName` doesn't? that's confusing :(

Reply:
OK yeah this isn't marvelous code:

```py
# OOTScene
    def sceneName(self):
        return self.name + "_scene"
```

Reply:
to explain that better, the file is called ``NAME_scene_cs_INDEX.c`` but the spec entry is called ``NAME_cs_INDEX.c``, so you get a file not found error when you compile the game

I used "_scene_" instead of the function as it's consistent with the other filenames earlier in the code